pages/pipelines/functions.rst

```diff
@@ -814,7 +814,7 @@ Example::
         rule "drop messages over 16383 characters"
         when
             has_field("message") AND
-            regex(to_string($message.message), "^.{16383,}$").matches == true
+            regex("^.{16383,}$", to_string($message.message)).matches == true
```

Comment:
I would prefer if you would include:

```
regex(pattern: "^.{16383,}$",value: to_string($message.message)).matches == true
```

this way it is clear and the order does not matter.

Reply:
I agree, this is cleaner. I'll make the changes.